docker/haal_centraal_brp/README.md

```diff
@@ -21,38 +21,54 @@ This brings up the service and you can now make API calls by using
 http://localhost:5010/haalcentraal/api/brp/personen. Of course these calls have to be according to
 the specification (https://brp-api.github.io/Haal-Centraal-BRP-bevragen/v2/redoc).
 
+Note: before running the related tests in the code, please make sure to apply all patches from
+_open-forms/patches_ to the test data. See below for more detailed instructions on how to create
+and apply patches.
+
 ### Test data
 
 Inside the container in `/app/Data/` directory you can find a json file with the initial test data
 that the API provides. This file is part of the docker container itself and it can be modified
-according to our needs. Trying to put unknown attributes/parameters will work as expected with a
-validation error (400).
+according to our needs by using patches. Trying to put unknown attributes/parameters will work as
+expected with a validation error (400).
 
-#### Please make sure you follow the correct steps when adding new/modifying test data:
+#### Adding a new patch to modify test data:
 
-- copy the _/app/Data/test-data.json_ file into your local machine
+- Copy the _/app/Data/test-data.json_ file onto your local machine. Ensure it is placed inside the
+  _open-forms_ folder to enable tracking by `git`.
 
-  `docker cp <container_id>:/app/Data/test-data.json /desired_path/test-data.json`
+  `docker cp <container_id>:/app/Data/test-data.json /path/to/open-forms/desired_path/test-data.json`
 
-- modify the data of the local file according to your needs
-- create a patch file (_some-change-bsnNumber.patch_) and save it to open-forms/patches directory by
-  using the last commit (with the changes ou made)
+- Commit the file to ensure changes are being tracked (this is only a temporary commit).
+- Modify the data of the local file according to your needs.
+- Commit the file again with the changes (this is only a temporary commit).
+- Create a patch file (_some-change-bsnNumber.patch_) and save it to _open-forms/patches_ directory by
+  using the last commit (with the changes you made).
 
-  `git format-patch -1`
+  `git format-patch -1 --stdout > /path/to/open-forms/patches/some-change-bsnNumber.patch`
```

Comment:
I don't think the second commit is necessary, some for of

```bash
git diff > change.patch
```
should be sufficient to record the changes into a patch file

A helper to apply the existing patches also wouldn't be wrong, you can use https://github.com/open-formulieren/open-forms/blob/archive/2.4.x/patches/apply.sh for reference :-)

In general, rather than doing these manual instructions I'd consider writing a bash script to automate this, it could live in the `bin` folder or in the `docker/haal_centraal_brp` folder

Reply:
Ok! @viktorvanwijk you can assign this to me if you prefer since this was my task.